kitsune/sumo/static/sumo/js/stores/UrlStore.es6.js

```diff
@@ -22,17 +28,37 @@ class _UrlStore extends BaseStore {
 const UrlStore = new _UrlStore();
 
 UrlStore.dispatchToken = Dispatcher.register((action) => {
-  let params, qs;
+  let params, pathProps, qs, urlPath;
   switch (action.type) {
+    case actionTypes.GET_PROPS_FROM_PATH:
+      pathProps = createDictFromPath(action.pathRegex, action.propNames);
+      urlData.pathProps = pathProps;
```

Comment:
Nit: This can probably just be `urlData.pathProps = createDictFromPath(action.pathRegex, action.propNames);`, unless there is a good reason to have it split like this.